app/styles/recap-situation.css

```diff
@@ -0,0 +1,66 @@
+.recap-situation {
+    border-top: none;
+    background-color: white;
+}
+
+.recap-situation h3 {
+    margin-top: 0;
+}
+
+.recap-situation .section-header {
```

Comment:
Replace the `section-header` class by `h4`, so we can get rid of this class and selector altogether. If this makes the single `h4` that doesn't have this class look bad, then just make it into a `h5`.